app/views/admin/deploy_groups/_environment_selector.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_tag '?', method: :get do %>
+  Environment: <%= select_tag :environment_id, options_for_select([['Matching', '']] + Environment.all.map { |e| [e.name, e.id] }, params[:environment_id]) %>
```

Comment:
Yeah true ... that's what matching was supposed to mean ... but we can be
clearer :)

On Wed, Jan 20, 2016 at 3:09 PM, Jon Moter <notifications@github.com> wrote:

> In app/views/admin/deploy_groups/_environment_selector.html.erb
> <https://github.com/zendesk/samson/pull/724#discussion_r50335998>:
>
> > @@ -0,0 +1,12 @@
> > +<%= form_tag '?', method: :get do %>
> > +  Environment: <%= select_tag :environment_id, options_for_select([['Matching', '']] + Environment.all.map { |e| [e.name, e.id] }, params[:environment_id]) %>
>
> How about the default value is the environment of the deploy group?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/724/files#r50335998>.
>